metricbeat/etc/fields.yml

```diff
@@ -8,7 +8,28 @@ defaults:
   ignore_above: 1024
 
 sections:
+  - ["general", "General event data"]
   - ["redis-info", "Redis Info Stats"]
+
+general:
+  type: group
+  description: >
+    General fields for each event
+  fields:
+    - name: response_time
```

Comment:
I really like the suggestion. I would go with `rtt` as it is shorter and I would say a common abbrevation? Ok going with nano seconds?